pkg/reconciler/instances/istio/istioctl/commander.go

```diff
@@ -0,0 +1,29 @@
+package istioctl
+
+import (
+	"os"
+	"os/exec"
+)
+
+//go:generate mockery -name=Commander
+// Commander for istioctl binary.
+type Commander interface {
+
+	// Install wraps istioctl installation command.
+	Install(istioCtlPath, istioOperatorPath, kubeconfigPath string) error
+}
+
+// DefaultCommander provides a default implementation of Commander.
+type DefaultCommander struct{}
+
+func (c *DefaultCommander) Install(istioCtlPath, istioOperatorPath, kubeconfigPath string) error {
+	cmd := exec.Command(istioCtlPath, "apply", "-f", istioOperatorPath, "--kubeconfig", kubeconfigPath, "--skip-confirmation")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
```

Comment:
Maybe worth to add also the STDERR + STDOUT output to the error (I haven't checked what the error object already includes, maybe it's already done like this by Golang)